packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -27,6 +27,20 @@ Terra.describeViewports('Form-Input', ['tiny', 'large'], () => {
     Terra.validates.element('invalid with incomplete');
   });
 
+  it('should display Form Input supported types', () => {
+    browser.url('/tests/terra-form-input/form-input/supported-types-input');
+    $('#root').moveTo({ xOffset: 0, yOffset: 700 }); // add to ensure consistent hover styles
+
+    Terra.validates.element('supported types');
+  });
+
+  it('should display Form Input unsupported types', () => {
+    browser.url('/tests/terra-form-input/form-input/unsupported-types-input');
+    $('#root').moveTo({ xOffset: 0, yOffset: 700 }); // add to ensure consistent hover styles
+
+    Terra.validates.element('unsupported types');
```

Comment:
I am wondering about the usefulness of having WDIO snapshots for the unsupported types - since they are already considered `unsupported` and will be dependent on browser changes, not terra changes. Plus they don't all fit in the capture area.  My suggestion would be to remove this test.

Reply:
Updated here https://github.com/cerner/terra-core/pull/3386/commits/451995dda70a0c87b88b1a69e7e802692ae84c2d.